mlcomp/tests/scripts/test_sagemaker_pipelines.sh

```diff
@@ -39,26 +39,47 @@ echo "Artifacts dir: ${artifacts_dir}"
 
 mlcomp_root="$script_dir/../.."
 components_root="$mlcomp_root/tests/components/sagemaker"
+pipelines_root="$mlcomp_root/tests/pipelines"
 model_filepath="${artifacts_dir}/sagemaker-kmeans-model.tar.gz"
 
+function pipeline_with_creds {
+    pipeline_name=$1
+    pipeline_filepath=${pipelines_root}/${pipeline_name}
+    local pipeline_with_creds_filepath="/tmp/${pipeline_name}"
+
+    cp ${pipeline_filepath} ${pipeline_with_creds_filepath}
+
+    region=$(grep region ~/.aws/config | awk '{print $3}')
+    aws_access_key_id=$(grep aws_access_key_id ~/.aws/credentials | awk '{print $3}')
+    aws_secret_access_key=$(grep aws_secret_access_key ~/.aws/credentials | awk '{print $3}')
+
+    sed -i'' -e "s/__REGION_PLACEHOLDER__/${region}/g" ${pipeline_with_creds_filepath}
```

Comment:
I strongly suggest using thee jinja2 package for performing search and replace in files.

Reply:
As far as I could see it is a python package. The current script is a bash script. I'm not sure it's really worth calling python or be dependent on this package.